docs/tutorials/go/audiobook/index.md

```diff
@@ -0,0 +1,971 @@
+This message is updated at each stage of the conversion process.
+
+With your Workflows and Activities in place, you can now write a [Worker application](https://docs.temporal.io/workers#worker-program).
+A [Worker](https://docs.temporal.io/workers) hosts your Activities and Workflows and polls a Temporal Service-provided Task Queue looking for work to do.
+
+## Create your Worker
+
+Add the following file contents to a new `worker/main.go` file:
+
+<!--SNIPSTART audiobook-project-go-worker-->
+[worker/main.go](https://github.com/temporalio/build-audiobook-go/blob/main/worker/main.go)
+```go
+package main
+
+import (
+    "log"
+    "os"
+    "strings"
+    "audiobook/app"
+
+    "go.temporal.io/sdk/client"
+    "go.temporal.io/sdk/worker"
+)
+
+const TaskQueue = "tts-task-queue"
+
+func main() {
+    bearerToken := os.Getenv("OPEN_AI_BEARER_TOKEN")
+    if bearerToken == "" {
+        log.Fatalln("Environment variable OPEN_AI_BEARER_TOKEN not found")
+    }
+
+    bearerToken = strings.TrimSpace(bearerToken)
+    bearerToken = strings.Map(func(r rune) rune {
+        if r >= 32 && r <= 126 { // Printable characters range
+            return r
+        }
+        return -1
+    }, bearerToken)
+
+    app.BearerToken = bearerToken
+
+    c, err := client.Dial(client.Options{})
+    if err != nil {
+        log.Fatalln("unable to create Temporal client", err)
+    }
+    defer c.Close()
+
+    w := worker.New(c, TaskQueue, worker.Options{})
+
+    w.RegisterWorkflow(app.TTSWorkflow)
+    w.RegisterActivity(app.ReadFile)
+    w.RegisterActivity(app.CreateTemporaryFile)
+    w.RegisterActivity(app.Process)
+    w.RegisterActivity(app.MoveOutputFileToPlace)
+
+    err = w.Run(worker.InterruptCh())
+    if err != nil {
+        log.Fatalln("unable to start Worker", err)
+    }
+}
+```
+<!--SNIPEND-->
+
+Every Worker polls [Task Queues](https://docs.temporal.io/workers#task-queue) to pick up Workflow and Activity Tasks.
+The Task Queue for this project is named `tts-task-queue`.
+You'll use this queue to submit conversion requests.
+Normally you'd build a dedicated full stack solution that submits requests to the Temporal Service and retrieves the results when they're complete.
+For this project, you issue your request through the command-line.
+
+A few more things about this code:
+
+- As a standalone application, this Worker has a `main` method.
+- The app starts by checking for an OpenAI bearer token and stores it as a static variable in the `app` package.
+- Next, it builds a [Client](https://docs.temporal.io/develop/go/temporal-clients), a class that can communicate with the Temporal service.
+- It uses the client and the Task Queue to create a new Worker, which registers the Workflow and Activities it can manage.
+- Finally, it runs the Worker, which starts polling for Tasks.
+
+Your complete project structure will now look like this:
+
+```sh
+.
+├── TTSActivities.go
+├── TTSWorkflow.go
+├── go.mod
+├── go.sum
```

Comment:
Very minor point, but `go.sum` shouldn't actually exist at this stage, since you haven't run `go mod tidy` to download dependencies yet

Reply:
Fixing!